app/methods/tag_adder.rb

```diff
@@ -0,0 +1,24 @@
+class TagAdder
+  def initialize(prediction:, string:)
+    @prediction = prediction
+    @string = string
+  end
+
+  def call
+    if string.present? && prediction.present?
+      tag_names.each do |tag_name|
+	      prediction.tag_names << tag_name
+      end && prediction.save
+    end
```

Comment:
This will add duplicates without complaining... but maybe the gem handles that, and maybe we don't care